<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apartemani_page</class>
 <widget class="QWidget" name="apartemani_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>اسانسور</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>قیمت پایه</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>دارد</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>بازگشت</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton_1">
     <property name="text">
      <string>اضافه کردن واحد</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>نام کاربری مدیر مسکن</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_1"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>آدرس</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>تعداد طبقه</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>مساحت کل</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>شناسه اپارتمان</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>ثبت آپارتمان</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>ندارد</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>پرونده اجاره</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>پرونده فروش</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
